hibernate-cache/src/test/java/org/infinispan/test/hibernate/cache/AbstractRegionAccessStrategyTest.java

```diff
@@ -184,39 +186,42 @@ protected void putFromLoadTest(final boolean useMinimalAPI, boolean isRemoval) t
 		final CountDownLatch writeLatch2 = new CountDownLatch(1);
 		final CountDownLatch completionLatch = new CountDownLatch(2);
 
+      CountDownLatch[] putFromLoadLatches = new CountDownLatch[2];
```

Comment:
it seems that you could have a separate variable for each latch rather than an array

Reply:
That would require quite a bit of rearchitecture of the code, which I wanted to avoid just to fix it.

If you separate into two variables, these variables need to be modified inside the lambda which is not allowed, so then you need to start creating separate Runnable classes that take all the state variables touched as parameters... etc.

TBH, the code is quite messy since it's calling for Callables rather this manual way of tracking exceptions...

Reply:
I see, sorry - the broken indentation got me confused.